packages/components/src/DataViewer/ModelViewer/Branch/__snapshots__/ModelViewerBranch.test.js.snap

```diff
@@ -18,9 +18,11 @@ exports[`ModelViewerBranch render ModelViewerBranch 1`] = `
       jsonpath="$"
       onToggle={[Function]}
       opened={false}
+      t={[Function]}
+      useCustomIcon={false}
```

Comment:
this was added because it's a default prop (and they are visible in snapshots now)